cmd/common-main.go

```diff
@@ -280,4 +281,23 @@ func handleCommonEnvVars() {
 		globalIsEnvWORM = true
 		globalWORMEnabled = bool(wormFlag)
 	}
+	skipCompressExtensions := os.Getenv("MINIO_SKIPCOMPRESS_EXTENSIONS")
+	skipCompressContentTypes := os.Getenv("MINIO_SKIPCOMPRESS_CONTENTTYPES")
```

Comment:
Can we change `MINIO_SKIPCOMPRESS_EXTENSIONS` -> `MINIO_SKIP_EXTENSIONS_TO_COMPRESS` & `MINIO_SKIPCOMPRESS_CONTENTTYPES` -> `MINIO_SKIP_CONTENTTYPES_TO_COMPRESS`

Reply:
It is better to have a common prefix perhaps better is 

MINIO_COMPRESS_SKIP_EXTENSIONS
MINIO_COMPRESS_SKIP_CONTENTTYPES